Userland/Libraries/LibWeb/HTML/Parser/StackOfOpenElements.cpp

```diff
@@ -68,9 +68,22 @@ bool StackOfOpenElements::has_in_list_item_scope(const FlyString& tag_name) cons
     return has_in_scope_impl(tag_name, list);
```

Comment:
On the Commit Title

Don't use "Fix ABC" when doing something more complex,
Explain what you are doing in the commit title, as per the contribution guidelines